src/utils/toPersondata.ts

```diff
@@ -39,6 +40,10 @@ export const toPersonData = (
       behandlerBerOmBistandUbehandlet: person.behandlerBerOmBistandUbehandlet,
       harArbeidsuforhetVurderAvslagUbehandlet:
         person.arbeidsuforhetVurderAvslagUbehandlet,
+      harFriskmeldingTilArbeidsformidling:
+        (person.friskmeldingTilArbeidsformidlingFom &&
+          isFuture(person.friskmeldingTilArbeidsformidlingFom)) ||
```

Comment:
Vi vil antagelig ha de i oversikten med friskmelding til arbeidsformding til veileder har avsluttet oppgave for vedtaket i Modia (uavhengig av om dato er tilbake tid eller ei), men det kan vi tilpasse senere :+1:

Reply:
Ja, vi får tilpasse etter diskusjonene på mandag.